tests/functional/cli_test.py

```diff
@@ -1,11 +1,14 @@
 from subprocess import run
 
+import pytest
+
 
 def test_help():
     """Test the dependabot-alerts --help command."""
     run(["dependabot-alerts", "--help"], check=True)
 
 
+@pytest.mark.xfail
```

Comment:
The cookiecutter generates `--help` and `--version` commands and functional and unit tests for those commands. This is partly because it's actually useful to have tests for those commands (they're useful commands after all, they should have tests) and partly because they exercise the whole mechanism of the CLI not just the actual help and version printing.

Unfortunately the `--version` command in the cookiecutter is broken in apps that have a required command line argument like `dependabot-alerts` does: `dependabot alerts hypothesis --version` works but `dependabot alerts --version` doesn't. See: <https://github.com/hypothesis/dependabot-alerts/issues/1>. I should fix this in the cookiecutter as a couple of our tools have this issue. For now I've just marked the test as xfail

Reply:
Fixed this